tsflex/features/segmenter/strided_rolling.py

```diff
@@ -142,17 +142,12 @@ def __init__(
 
         # 4. Check the sparsity assumption
         if not self.approve_sparsity:
-            qs = [0, 0.1, 0.5, 0.9, 1]
             for container in self.series_containers:
-                series_idx_stats = np.quantile(
-                    container.end_indexes - container.start_indexes, q=qs
-                )
-                q_str = ", ".join([f"q={q}: {v}" for q, v in zip(qs, series_idx_stats)])
                 # Warn when min != max
-                if not all(series_idx_stats == series_idx_stats[-1]):
+                if np.ptp(container.end_indexes - container.start_indexes) != 0:
```

Comment:
Calculates difference between min and max.
Is significantly faster than the previous implementation (that used `np.quantile`)

In this benchmark `idx = np.arange(100_000)`
![image](https://user-images.githubusercontent.com/18898740/164970792-d74539cf-8b19-4dae-8852-ec4429ef82a0.png)

Reply:
nice one! 🔥